.github/workflows/build-common.yml

```diff
@@ -18,6 +18,25 @@ permissions:
   contents: read
 
 jobs:
+  spotless:
```

Comment:
this splits spotless out into separate job so if it fails you can still see if the other jobs succeed or fail

probably not that important anymore with the autospotless automation